client-lib/src/main/java/com/ibm/ws/repository/connections/RepositoryConnection.java

```diff
@@ -37,14 +38,14 @@
 
     /**
      * Checks if the repository is available.
-     * 
```

Comment:
In this commit, there are some places where we're adding spaces inside comments and others where we're taking them out.

Are the formatting rules just causing them to flip-flop?

If so we should sort that out, but for now, resaving all the files with these changes in should remove them.